coops-ui/src/assets/styles/base.scss

```diff
@@ -288,6 +289,27 @@ ul.list {
   font-weight: 500;
 }
 
-.inline-div{
-  display: inline
+.genErr {
+  font-size: 0.9rem;
+}
+
+.white-background {
+  background-color: white !important;
```

Comment:
What are we overriding here, if this is the base theme style?

Reply:
Some v-alert components are by default transparent (on a gray app background), so we use this style in a couple of places to make them have a white background and stand out a bit.

Reply:
right on, i've noticed a lot of styling issues are usually a clash between the component libraries and the designers goals 😄  Thanks for the info!